website/bin/exrmetrics.rst

```diff
@@ -0,0 +1,185 @@
+..
+  SPDX-License-Identifier: BSD-3-Clause
+  Copyright Contributors to the OpenEXR Project.
+
+exrmetrics
+##########
+
+::
+   
+    exrmetrics [options] infile [infile2...] [-o outfile]
+
+Description
+-----------
+
+Read an OpenEXR image from infile, write an identical copy to outfile reporting time taken to read/write and file sizes.
+
+Options:
+--------
+
+.. describe:: -o file
+
+   File to write to. If no file specified, uses a memory buffer.
+
+   Note: file may be overwritten multiple times during tests
+
+
+.. describe:: -p n
+
+   Part number to copy, or ``all`` for all parts. Default is ``all``.
+              
+
+.. describe:: -m
+
+   Set to multi-threaded (system selected thread count).
+
+.. describe:: -t n
+
+   Use ``n`` threads for processing files. Default is single / no threads.
```

Comment:
Looking at this, the `single / no threads` is a bit confusing. I wonder if it would be better to say:

Use a pool of `n` worker threads for processing files. Default is single threaded (no thread pool)

Reply:
done.